README.md

```diff
@@ -8,12 +8,15 @@ the namespace either at creation time or at any later point:
 | -------- | -------------------------- | -------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
 | Hard     | `ci.openshift.io/ttl.hard` | `time.Duration`-formatted string | The time after namespace creation when the namespace should be reaped.                                                 |
 | Soft     | `ci.openshift.io/ttl.soft` | `time.Duration`-formatted string | The time after `Pod` termination where, if no other `Pod`s are `Pending` or `Running`, the namespace should be reaped. |
+| Active   | `ci.openshift.io/active`   | `time.Time`	 -formatted string | The last active marker is optional and stored as a RFC3339-formatted date.												|
```

Comment:
The table shows you TTL types... this is not a TTL type. I think we should literally just have this in the description for the soft TTL.

```
 | TTL Type |       Annotation Key       |         Annotation Value         |                                                       Description                                                      |
| -------- | -------------------------- | -------------------------------- | ---------------------------------------------------------------------------------------------------------------------- |
| Hard     | `ci.openshift.io/ttl.hard` | `time.Duration`-formatted string | The time after namespace creation when the namespace should be reaped.                                                 |
| Soft     | `ci.openshift.io/ttl.soft` | `time.Duration`-formatted string | The time after `Pod` termination or last active marker where, if no other `Pod`s are `Pending` or `Running`, the namespace should be reaped. The last active marker is optional and stored as a RFC3339-formatted date in the `ci.openshift.io/active` annotation. |
```